back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -155,6 +157,14 @@ def create
     save_options[:context] = :publication if params.dig(:idea, :publication_status) == 'published'
     ActiveRecord::Base.transaction do
       if input.save save_options
+        params_for_file_upload_fields.each do |key, params_for_files_field|
+          idea_file = input.idea_files.create!(
+            name: params_for_files_field['name'],
+            file: params_for_files_field['content']
+          )
+          input.custom_field_values[key] = idea_file.id
```

Comment:
I notice that the custom field values of ideas are included in templates. I don't expect failures here, but the ID references will not work. Should we do something about this?

Reply:
That is indeed something that we have to discuss and solve.